Tests/NMAImageFetchTests/NMAImageFetchTests.swift

```diff
@@ -89,8 +90,8 @@ final class NMAImageFetchTests: XCTestCase {
         waitForExpectations(timeout: 1)
     }
 
-    func testImageNotFound() {
-        let url = URL(string: "https://www.nomasystems.com/404.png")!
+    func testImageNotFound() throws {
+        let url = try XCTUnwrap(URL(string: "https://github.com/nomasystems/nimagefetch-ios/blob/main/404.png"))
```

Comment:
"https://www.nomasystems.com/404.png" was returning a 403 so I changed it to get a real 404